webpack/localization-plugin/src/utilities/ResxReader.ts

```diff
@@ -245,26 +296,45 @@ export class ResxReader {
   }
 
   private static _logErrorWithLocation(
-    options: IResxReaderOptions,
+    options: IResxReaderOptionsInternal,
     message: string,
     element?: XmlElement | XmlDocument
   ): void {
     if (element) {
-      options.logFileError(message, options.resxFilePath, element.line, element.position);
+      options.loggingFunctions.logFileError(message, options.resxFilePath, element.line + 1, element.column + 1);
     } else {
-      options.logFileError(message, options.resxFilePath);
+      options.loggingFunctions.logFileError(message, options.resxFilePath);
     }
   }
 
   private static _logWarningWithLocation(
-    options: IResxReaderOptions,
+    options: IResxReaderOptionsInternal,
     message: string,
     element?: XmlElement | XmlDocument
   ): void {
     if (element) {
-      options.logFileWarning(message, options.resxFilePath, element.line, element.position);
+      options.loggingFunctions.logFileWarning(message, options.resxFilePath, element.line + 1, element.column + 1);
     } else {
-      options.logFileWarning(message, options.resxFilePath);
+      options.loggingFunctions.logFileWarning(message, options.resxFilePath);
     }
   }
-}
\ No newline at end of file
+
+  private static _logWithLocation(
+    loggingFn: (message: string) => void,
+    message: string,
+    filePath: string,
+    line?: number,
+    position?: number
+  ): void {
+    let location: string;
+    if (position !== undefined) {
+      location = `${filePath}(${line},${position})`;
```

Comment:
Does this schema work with Ctrl+Click in VSCode? If not, would be a nice-to-have to switch to one that does.

Reply:
Yeah works fine in VSCode.